pages/outreachy.md

```diff
@@ -54,8 +57,56 @@ Interested in becoming a mentor? Contact one of the coordinators!
 
 ## Available tasks
 
-Coming soon. At this moment, we recommend looking at the following issue trackers for smaller tasks that might
-give a sense of what's coming:
+### Instrumentation for mobile applications
```

Comment:
Doing mobile instrumentation efficiently is actually a non-trivial engineering project. For example, our mobile team has a whole _platform_ sub-team that built a _performance SDK_ that does clever things like gauging the available network bandwidth from the phone and sending monitoring data via unused bandwidth only.

Reply:
Agree that to provide a production quality solution will be non-trivial, but this type of project has been popular with GSoC students on Hawkular, and the expectation would only be to provide an initial proof of concept with demonstrator. If it then looks promising, further performance work could be done by more experienced contributors?